File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hashmap={}
        for i,num in enumerate(nums):
            comp=target-num
            if comp in hashmap:
                return [hashmap[comp],i]
            hashmap[num]=i
        

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let hashmap={}
    for(let i=0;i<nums.length;i++){
        comp=target-nums[i]
        if(comp in hashmap){
            return [hashmap[comp],i]
        }
        hashmap[nums[i]]=i

    }
    
};